components/network/openafs/openafs.p5m

```diff
@@ -0,0 +1,329 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Aurelien Larcher
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# Kernel modules
+<transform file path=kernel/drv/afs \
+    -> set action.hash usr/lib/openafs/libafs.o >
+file path=kernel/drv/afs
+
+<transform file path=kernel/drv/$(MACH64)/afs \
+    -> set action.hash usr/lib/openafs/libafs64.o >
+file path=kernel/drv/$(MACH64)/afs pkg.linted.userland.action001.2=true
+
+# Configuration
+file files/afs.rc     path=/etc/init.d/afs mode=0555
```

Comment:
@pyhalov I have SMF manifests from Mathias Feiler but I am not sure how they should be adapted.